src/frontend/pages/chat/chat.js

```diff
@@ -186,6 +186,9 @@ export class Chat extends LitElement {
                         "info",
                         2000
                       )}
+                    @edit-message=${(event) => {
+                      this.editMessage(event);
```

Comment:
Puoi riscriverlo semplicemente come @edit-message=${this.editMessage}, l'evento viene passato di default